### DISCUSSION TITLE
Alguém interessado em criar um grupo de estudo de Front-End?

### DISCUSSION THREAD

**Discussion by danielGalvao** [frontendbr/forum#338] [Discussão]
Estava pensando em criar um grupo tipo Meetup, porém online via Hangouts ou outro app... 
A idéia era ter temas e hands on! 
O ideal é juntar umas 4 pessoas para discutir e fazer código ao vivo e 1 ou 2 ir tirando dúvida nos comentários dos espectadores ao vivo.
Acham válido? Digo isso pois nem sempre posso ou tenho como me locomover até um Meetup e sendo online acho que traria mais gente de todo lugar para discutir e praticar sobre os diversos assuntos do Front-end.

**Comment by woliveiras** [comment]
Acho super válido @danielGalvao 

Eu estudava assim com uma galerinha pelas interwebs! Valeu muito a pena.

Até comecei um esquema pra quem busca grupos de estudos aqui: https://github.com/training-center/study-groups

**Reply by Nataliglima** [reply]
algum grupo de estudo  aí para front end 2023????

**Comment by ranosi** [comment]
Pow eu acho uma boa também. Se realmente for acontecer notifica aí, pode
contar comigo.

2016-10-25 17:25 GMT-02:00 William Oliveira notifications@github.com:

> Acho super válido @danielGalvao https://github.com/danielGalvao
> 
> Eu estudava assim com uma galerinha pelas interwebs! Valeu muito a pena.
> 
> Até comecei um esquema pra quem busca grupos de estudos aqui:
> https://github.com/training-center/study-groups
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> https://github.com/frontendbr/forum/issues/338#issuecomment-256149476,
> or mute the thread
> https://github.com/notifications/unsubscribe-auth/AV1hh7M-kUESNh325GEHALE9xhR7Mz-mks5q3lehgaJpZM4KgWrH
> .

## 

Grande abraço,

_Rafael Nogueira Silva_ - @rafaelnogueira

**Comment by omarkdev** [comment]
Bem interessante, acho que podemos escolher um tema a cada mês (Seja PWA, VueJS, React) e fazer algo com ele, depois colocar num repositório e explicar aquilo, assim ajudando pessoas no futuro que querem tentar também.

**Comment by alexsandro-xpt** [comment]
Oh eu também achei legal... vamos sim.... me chama!!

**Comment by williamcosta** [comment]
Se rolar estou dentro!

**Comment by Kirmayrtomaz** [comment]
Eu também quero

2016-10-26 11:57 GMT-04:00 William Costa notifications@github.com:

> Se rolar estou dentro!
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> https://github.com/frontendbr/forum/issues/338#issuecomment-256394205,
> or mute the thread
> https://github.com/notifications/unsubscribe-auth/AJO8tNchmPhG58X0rYmsl5V7N93XMtf0ks5q33hqgaJpZM4KgWrH
> .

## 

Kirmayr Tomaz
Bacharel em Sistemas de Informação.
Universidade Federal do Amazonas

**Comment by hugoalmeidahh** [comment]
Eu to dentro também...

**Comment by raphaelmelo** [comment]
Tenho interesse também, vamos colocar em pratica essa ideia. Sou pouco auto-didata, por isso, muitas vezes me perco. Seria bacana alguém auxiliando.

**Comment by raulfernando** [comment]
Opa me interesso tbm

**Comment by ranosi** [comment]
Também tenho esse probleminha de me perder as vezes, ainda mais que preciso fazer uma duzia de coisa por mês. O CTgroup no Slack pode ser usado pra trocarmos ideias sobre isso?

**Comment by ericktatsui** [comment]
To nessa. É bem motivador estudar com outras pessoas.

**Comment by danielGalvao** [comment]
Legal pessoal!! Vamos amadurecendo então.
Vamos tentar ser mais organizado.
Além de ter interesse vamos responder as seguintes perguntas:
1 - Quais temas tem interesse em aprender (4 no máximo)
2 - Quais temas tem interesse em ensinar (4 no máximo)
3 - Qual ferramenta gostaria de usar para reunir o grupo e definir as coisas (slack, hangouts, etc)
4 - Qual melhor dia/horário para você participar do grupo de estudo
Abraços

**Comment by diogokobbi** [comment]
Boa tarde a todos, peço desculpas se atrapalho a discussão. Qual nível de conhecimento é necessário para participar do grupo de estudos? Se for avançado, os demais podem participar como ouvintes?

**Comment by woliveiras** [comment]
Nosso Slack pode ser usado pra combinar tudo sim @ranosi 😄

**Comment by danielGalvao** [comment]
@woliveiras qual é o Slack? pode passar? 

@diogokobbi  Não teria nivelamento, acho que até o que tenha menos conhecimento sobre um tema deva participar no hands on para também receber ajuda diretamente

**Comment by diogokobbi** [comment]
@danielGalvao OK! Gostaria de participar também.

**Comment by woliveiras** [comment]
Opa, só colocar o email aqui, @danielGalvao: https://ctgroups.herokuapp.com/

:metal: :metal: :metal:

**Comment by Lucasktrindade** [comment]
Acho maneiro a ideia, toparia participar e ajudar a galera. Não seria melhor usar o Slack do Front End Br já?

**Comment by bernardodiasc** [comment]
http://frontendbrasil-slack.herokuapp.com/ (slack deste forum) 🔥 🔥 🔥 🔥 🔥

**Comment by woliveiras** [comment]
Vocês que mandam @Lucasktrindade 

Eu indiquei o CT por já ser um projeto totalmente voltado a estudos e mentoria. 😄

**Comment by alexsandro-xpt** [comment]
1 - Quais temas tem interesse em aprender (4 no máximo)
Tooling
React+Redux+GraphQL
CSS 3(Resposive, e tecnicas de projetos modular CSS, SASS)
Transformar layout em html( discutir novas técnicas e ferramentas )

2 - Quais temas tem interesse em ensinar (4 no máximo)
Webpack
RequireJS
Angular

3 - Qual ferramenta gostaria de usar para reunir o grupo e definir as coisas (slack, hangouts, etc)
slack ou telegram discutir e hangouts para encontro ao vivo.

4 - Qual melhor dia/horário para você participar do grupo de estudo
Qualquer dia a noite a partir das 20:00 e aos finais de semana, a tarde.

**Comment by santosluiz** [comment]
Estou dentro também! É só chamar!!

**Comment by dionarodrigues** [comment]
To dentro tb! =D

**Comment by kelciocajueiro** [comment]
Também tenho interesse!

**Comment by adeonir** [comment]
Também to dentro.

**Comment by pedrobritto** [comment]
Também tenho interesse!

**Comment by lucascdsilva** [comment]
\o/

**Comment by ldrbraga** [comment]
Bom dia, galera! 

Tá rolando algum grupo de estudo?

**Comment by AmandaFonseca** [comment]
Quero participar

**Comment by renanlara** [comment]
Genial a ideia, tô dentro!
